ios/brave-ios/Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -192,6 +192,14 @@ public class BraveRewards: PreferencesObserver {
     }
   }
 
+  // MARK: - Brave Ads History
+
+  /// Clear Brave Ads History.
+  @discardableResult
+  func clearAdsHistory() async -> Bool {
+    return await ads.clearHistory()
+  }
```

Comment:
This will run off the UI task runner/thread, if it needs to be run on main please add `@MainActor` alongside `@discardableResult`

Reply:
This is specifically to clear the history of Brave Ads, as these ads can still run even if Rewards are disabled for search result ads.

Reply:
Assume this was meant to be commented on the above review comment but I just meant will a user even understand what this means given that we dont really have the words "Brave Ads" written anywhere in the app

Reply:
Ah yes, let me move this to that thread. Thanks

Reply:
@kylehickinson maybe we could add a description for this switch? Thanks

Reply:
added @MainActor